analysis/analysis-engine.go

```diff
@@ -6,7 +6,10 @@
 package analysisengine
 
 import (
-	"github.com/letsencrypt/boulder/log"
+	blog "github.com/letsencrypt/boulder/log"
```

Comment:
Maybe rename this library 'logging'? The name blog makes me confused with weblogging.